### DISCUSSION TITLE
Compare option not available

### DISCUSSION THREAD

**Discussion by nfbarreto** [plausible/analytics#3003] [Self-Hosted Support] [answered]
Hi

I'm self-hosting Plausible Analytics and noticed that the "Compare" option is not available.

Is this something missing in the self-hosted option? I looked into the configurations and couldn't find anything talking about it. It is a recent installation (about 3 days), but I've imported GA data. 

Thank you.

Self-hosted:
![image](https://github.com/plausible/analytics/assets/32324161/594fb415-9c89-4752-be0b-d0b1309a79d6)

Trial:
![image](https://github.com/plausible/analytics/assets/32324161/c6ef9a03-ee23-47e2-81e6-b128551ef7f1)

**Comment by ruslandoga** [comment, marked as answer]
👋 @nfbarreto 

What version are you using?

This feature was added in [`v2.0.0-rc.1`](https://github.com/plausible/analytics/discussions/2981)

**Reply by nfbarreto** [reply]
I think I'm using 1.5.1. I followed the installation documentation. 

![image](https://github.com/plausible/analytics/assets/32324161/5e96ac4a-e97d-4b37-bf1c-97ed8ab149a8)

**Reply by ruslandoga** [reply]
You can check the image tag on `plausible` in your `docker-compose.yml`

**Reply by nfbarreto** [reply]
Yes, V1.5.1. is the one I have.

![image](https://github.com/plausible/analytics/assets/32324161/88378afd-cb7a-481a-8eea-2642667d5ba8)

I used the template file at https://github.com/plausible/hosting/blob/master/docker-compose.yml without changing anything. 

I will change it to consider "latest" or "v2.0.0-rc.1".

Thank you for the help.

**Reply by ruslandoga** [reply]
Note that `latest` is `v1.5.1`. If you change it to `v2.0.0-rc.1`, please follow the upgrade steps outlined in https://github.com/plausible/analytics/discussions/2981

**Reply by nfbarreto** [reply]
Got it! Thank you for the help, @ruslandoga!
